qcodes_contrib_drivers/drivers/RohdeSchwarz/SMW200A.py

```diff
@@ -0,0 +1,2313 @@
+                           Use sweep_start and sweep_stop or
+                           use sweep_center and sweep_span
+                           to define the sweep range.
+                           """)
+
+        self.add_parameter('sweep_stop',
+                           label='Stop frequency of the sweep',
+                           set_cmd='SOUR{}:FREQ:'.format(self.chnum)+'STOP {}',
+                           get_cmd='SOUR{}:FREQ:STOP?'.format(self.chnum),
+                           get_parser=float,
+                           vals=vals.Numbers(300e3, maxfreq),
+                           unit='Hz',
+                           docstring="""
+                           The stop frequency of the sweep.
+                           Use sweep_start and sweep_stop or
+                           use sweep_center and sweep_span
+                           to define the sweep range.
+                           """)
+
+        # Parameter for the LOSCILLATOR
+        self.add_parameter('losc_input',
+                           label='LOscillator input frequency',
+                           set_cmd=False,
+                           get_cmd='SOUR{}:FREQ:LOSC:INP:FREQ?'.format(self.chnum),
+                           get_parser=float,
+                           unit='Hz',
+                           docstring="(ReadOnly) Read the LOscillator input frequency.")
+
+        lomode = vals.Enum('INT', 'EXT', 'COUP', 'ECO', 'BOFF', 'EBOF', 'AOFF')
+        self.add_parameter('losc_mode',
+                           label='LOscillator mode',
+                           set_cmd='SOUR{}:FREQ:LOSC:'.format(self.chnum)+'MODE {}',
+                           get_cmd='SOUR{}:FREQ:LOSC:MODE?'.format(self.chnum),
+                           vals=lomode,
+                           docstring="""
+                           The LOscillator mode. Valid values are:
+                           'INT'  = A&B Internal / Internal (one path instrument) - Uses the internal oscillator signal in both paths.
+                           'EXT'  = A External & B Internal (one path instrument) - Uses an external signal in path A. B uses its internal signal.
+                           'COUP' = A Internal & A->B Coupled - Assigns the internal oscillator signal of path A also to path B.
+                           'ECO'  = A External & A->B Coupled - Assigns an externally supplied signal to both paths.
+                           'BOFF' = A Internal & B RF Off - Uses the internal local oscillator signal of path A, if the selected frequency exceeds the maximum frequency of path B.
+                           'EBOF' = A External & B RF Off - Uses the LO IN signal for path A, if the selected RF frequency exceeds the maximum frequency of path B.
+                           'AOFF' = A RF Off & B External - Uses the LO IN signal for path B, if the selected RF frequency exceeds the maximum frequency of path A.
+                           """)
```

Comment:
Some formatting needed for this docstring.

Reply:
What exactly do you mean with formatting?

I've changed the indentation and replaced the equals sign with colons. Is that what you've meant?